docker/zk-environment/20.04_amd64_cuda_11_8.Dockerfile

```diff
@@ -0,0 +1,183 @@
+FROM ubuntu:20.04@sha256:3246518d9735254519e1b2ff35f95686e4a5011c90c85344c1f38df7bae9dd37 as base
+
+WORKDIR /usr/src/zksync
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install required dependencies
+RUN apt-get update && apt-get install -y \
+    cmake \
+    make \
+    bash \
+    git \
+    openssl \
+    libssl-dev \
+    gcc \
+    g++ \
+    curl \
+    pkg-config \
+    software-properties-common \
+    jq \
+    openssh-server \
+    openssh-client \
+    wget \
+    vim \
```

Comment:
Didn't expect to see this lil fellow here.

Reply:
@popzxc can we review what was changed? i intentionally do not want to change anything in the old images)

Reply:
You may rightfully complain when unrelated comments are accompanied by the sad red icon at the top, but not when it's just a comment. It could've been a gif with a cute kitten instead, would you be as angry then?